tasks/kernel_matrix_testing/env-setup.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-set -eo xtrace
+set -e
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why `-o` is removed? (context: I don't know what the `-e` option does)

Reply:
This script is not run in the context of CI, if that changes whether this needs `-x` removed.